arguments_parsing.py

```diff
@@ -14,10 +19,10 @@ def parse_connection_arguments():
     connection_parameters = {}
     config_file_path = f"{CONFIG_PATH}/{CONNECTION_CONFIG_NAME}"
     if not os.path.isfile(config_file_path):
-        print(f"Error, there is no file {config_file_path}")
+        logger.critical(f"Error, there is no file {config_file_path}")
         raise Exception(f"Error, there is no file {config_file_path}")
 
-    with open() as connection_config_file:
+    with open(config_file_path) as connection_config_file:
```

Comment:
No exception handling for case of fail during file opening.

Reply:
resolved. Add some exceptions